monolith/views/restaurants.py

```diff
@@ -45,3 +45,21 @@ def _like(restaurant_id):
     else:
         message = "You've already liked this place!"
     return _restaurants(message)
+
+@restaurants.route("/create_restaurant")
```

Comment:
This was already implemented in https://github.com/GreyTeam2020/GoOutSafe_Primer2020/pull/5

Can you remove this change inside this file? I will join my change with yours